contracts/Bridge.sol

```diff
@@ -404,16 +407,16 @@ contract Bridge is Pausable, AccessControl, SafeMath {
         bytes32 dataHash = keccak256(abi.encodePacked(handler, data));
         Proposal storage proposal = _proposals[nonceAndID][dataHash];
 
-        require(proposal._status != ProposalStatus.Inactive, "proposal is not active");
-        require(proposal._status == ProposalStatus.Passed, "proposal already transferred");
-        require(dataHash == proposal._dataHash, "data doesn't match datahash");
+        require(proposal._status != ProposalStatus.Inactive, "Proposal is not active");
```

Comment:
unrelated to your PR but perhaps we could remove the inactive check? And just realized the require string for Proposal already transferred is incorrect. Should be along the lines of "Can't execute proposal without Passed status" but hopefully under 32 characters